### PR TITLE
Replace bigint casting with bigint literal

### PR DESCRIPTION
**Pull request by NullSoldier**
## Summary

There's no reason to wrap bigints in the bigint type. You can just use literals. This only replaces some places.

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/strategy.test.slow.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ describe('Demonstrate the Sapling API', () => {
         workerPool: new WorkerPool(),
         consensus: new TestnetConsensus(consensusParameters),
       })
-      const minersFee = await strategy.createMinersFee(BigInt(0), 0, key.spendingKey)
+      const minersFee = await strategy.createMinersFee(0, 0, key.spendingKey)
```
Comment (hughy) [inline ironfish/src/strategy.test.slow.ts:216]:
```suggestion
      const minersFee = await strategy.createMinersFee(0n, 0, key.spendingKey)
```

Missing `n` to make this a bigint literal

File: ironfish/src/primitives/target.ts [hunk 2]
```diff
@@ -7,28 +7,25 @@ import { BigIntUtils } from '../utils/bigint'
...
-)
+const MAX_TARGET = 883423532389192164791648750371459257913741948437809479060803100646309888n
 
 /**
  *  Maximum number to represent a 256 bit number, which is 2**256 - 1
  */
-const MAX_256_BIT_NUM = BigInt(
-  '115792089237316195423570985008687907853269984665640564039457584007913129639935',
-)
+const MAX_256_BIT_NUM =
```
Comment (danield9tqh) [inline ironfish/src/primitives/target.ts:21]: Here do you want to do this for readability?
```javascript
const MAX_256_BIT_NUM = 2n ** 256n - 1n
```

Comment (danield9tqh) [reply ironfish/src/primitives/target.ts:21]: could do in another change too

**Review by dguenther — APPROVED** [review]

File: ironfish/src/primitives/target.ts (hunk 2 above)
Comment (dguenther) [reply ironfish/src/primitives/target.ts:21]: I think that's clean, but I'd recommend doing it in another PR just to avoid the back-and-forth

**Pull request merged**